src/api/wayfire/view.hpp

```diff
@@ -168,6 +177,24 @@ class view_interface_t : public surface_interface_t, public wf::object_base_t
      */
     virtual wlr_box get_bounding_box();
 
+    last_output_info_t last_output_info;
+
+    /**
```

Comment:
This would be better implemented with custom data on the view, see get_data/store_data/etc in object.hpp

Reply:
This way, the struct also doesn't have to be in the view.hpp header file and can be kept private to the plugin

Reply:
Done.